client/src/core/world-manager.js

```diff
@@ -69,6 +69,9 @@ class WorldManager {
     ].map((c) => c / 255.0));
     this.engine3d.setSkyColorSpinning(false);
 
+    if (data.enableFog) {
+      this.engine3d.setFog(data.fogColor, data.fogMinimum, data.fogMaximum);
+    }
```

Comment:
You may want to check for the else case to disable/reset the fog from the engine3d

Reply:
or you can disable it on world unload